tmk_core/common/print.h

```diff
@@ -180,18 +169,6 @@ extern "C"
 #        define print_bin_reverse8(i) xprintf("%08b", bitrev(i))
 #        define print_bin_reverse16(i) xprintf("%016b", bitrev16(i))
 #        define print_bin_reverse32(i) xprintf("%032lb", bitrev32(i))
-/* print value utility */
-#        define print_val_dec(v) xprintf(#        v ": %u\n", v)
-#        define print_val_decs(v) xprintf(#        v ": %d\n", v)
-#        define print_val_hex8(v) xprintf(#        v ": %X\n", v)
-#        define print_val_hex16(v) xprintf(#        v ": %02X\n", v)
-#        define print_val_hex32(v) xprintf(#        v ": %04lX\n", v)
-#        define print_val_bin8(v) xprintf(#        v ": %08b\n", v)
-#        define print_val_bin16(v) xprintf(#        v ": %016b\n", v)
-#        define print_val_bin32(v) xprintf(#        v ": %032lb\n", v)
-#        define print_val_bin_reverse8(v) xprintf(#        v ": %08b\n", bitrev(v))
-#        define print_val_bin_reverse16(v) xprintf(#        v ": %016b\n", bitrev16(v))
-#        define print_val_bin_reverse32(v) xprintf(#        v ": %032lb\n", bitrev32(v))
```

Comment:
I would propose that rather than remove these, they be fixed to have the correct field widths.
I know that they aren't used in the merged code-base, but I do occasionally use them when debugging.

Note that as preprocessor macros, defining them has no impact on firmware size (unless they are used).